app/models/manageiq/providers/redhat/infra_manager/provision/state_machine.rb

```diff
@@ -33,15 +33,19 @@ def poll_clone_complete
   end
 
   def customize_destination
-    if destination_image_locked?
-      _log.info("Destination image locked; re-queuing")
-      requeue_phase
+    if source.template?
+      if destination_image_locked?
+        _log.info("Destination image locked; re-queuing")
+        requeue_phase
+      else
+        message = "Starting New #{destination_type} Customization"
```

Comment:
Why we use capital letter for "new" and "customization"?

Reply:
developer excuses #211: "it was like that before"

seems like other providers (vmware and microsoft) have the exact message (with the capitals)